File: codes/dotnet/day-5/InheritanceDemo/InheritanceDemo/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InheritanceDemo
{
    class Person
    {
        protected string name;
        protected string department;

        public Person() { }

        public Person(string name, string departmentName)
        {
            this.name = name;
            this.department = departmentName;
        }

        public string Name
        {
            set => this.name = value;
            get => this.name;
        }

        public string Department
        {
            set => this.department = value;
            get => this.department;
        }

        //public string PrintInfo()
        //{
        //    return $"{name}, {department}";
        //}
    }
}

File: codes/web/day-2/single-page-app/src/models/person.ts
class Person {
    
    private id: number;
    private name: string;
    private salary: number;

    constructor(id: number, name: string, salary: number) {
        this.id = id
        this.name = name
        this.salary = salary
    }

    public get Id(): number {
        return this.id
    }
    public set Id(val: number) {
        this.id = val
    }
    public set Name(val:string){
        this.name = val
    }
    public get Name():string{
        return this.name
    }
    public set Salary(val:number){
        this.salary = val
    }
    public get Salary():number{
        return this.salary
    }
}